docs/userGuide/syntax/includes.md

```diff
@@ -61,6 +61,62 @@ The `<include>` mechanism can be used inside any MarkBind source file (even insi
 </span>
 </include>
 
+<panel header="Markdown not rendered when including a variable">
+
+When trying to include a variable, markdown is not rendered.
+
+To fix this, either replace the markdown with html tags if possible or replace the `include` with the fragment.
+
+{{ icon_example }}
+
+<box>
+
+Include fragment in file "index.md":
+```html 
+<variable id="fragment-id">
+  
+  ### I am a header
+</variable>
+```
+
+Code that uses include fragment:
+```html
+<include src="index.md#fragment-id" />
+```
```

Comment:
Something is not right here. I don't think variables are meant to be used as include fragments in the first place.  Although it would be nice 'trick' to be able to do so, I don't think it is an advertised feature. Variable values are inserted using the `{{ v_name }}` instead. Does our UG say variables can be included in this way?

![image](https://user-images.githubusercontent.com/1673303/229336932-1893c209-0e76-4350-97d1-bab56195281d.png)

Reply:
Oh I see I see. In that case, I will close this PR and the corresponding issue :)